### PR TITLE
fix: print sonarqube report

### PR DESCRIPTION
**Pull request by brendaHuang**
#447

### CODE REVIEW

**Comment by brendaHuang** [comment]
Screenshot 2023-01-30 at 2 14 38 PM

**Comment by brendaHuang** [comment]
[Uploading migrate-report-for-sonarqube.txt…]()

**Comment by brendaHuang** [comment]
[migrate-report-for-sonarqube.pdf](https://github.com/rehearsal-js/rehearsal-js/files/10540493/migrate-report-for-sonarqube.pdf)

File: packages/cli/src/helpers/report.ts [hunk 1]
```diff
@@ -36,7 +36,7 @@ export function generateReports(
       reportPath = resolve(outputPath, `${reportBaseName}.json`);
       formatter = jsonFormatter;
     } else if (format === 'sonarqube') {
-      reportPath = resolve(outputPath, `${reportBaseName}.json`);
+      reportPath = resolve(outputPath, `${reportBaseName}-for-sonarqube.json`);
```
Comment (chadhietala) [inline packages/cli/src/helpers/report.ts:39]:
```suggestion
      reportPath = resolve(outputPath, `${reportBaseName}.sonarqube.json`);
````

Comment (lynchbomb) [reply packages/cli/src/helpers/report.ts:39]: ^^^ this

**Review by egorio — APPROVED** [review]

**Pull request merged**